package/CHANGELOG

```diff
@@ -13,7 +13,11 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
+<<<<<<< HEAD
```

Comment:
Conflicts are not fixed here.

Reply:
... actually, not sure if it was my fault, but I had a bad conscience and fixed it.